### PR TITLE
[DOC-2869] Update images

### PR DESCRIPTION
**Pull request by SmritiSatya**
Change 'Chaos Access' to 'Chaos Infrastructure' in images 
Replace 'Chaos Access' with 'Chaos Infrastructure' in the architecture documentation
Signed-off-by: Smriti S <smriti.satyanarayana@harness.io>

### CODE REVIEW

**Review by SudheendraKatte — DISMISSED** [review]

File: docs/chaos-engineering/technical-reference/architecture.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The diagram below gives a peek into the HCE architecture.
 
 ## Harness control plane
 
-**Harness control plane** is the single source to collaboratively create, schedule, and monitor chaos experiments. It comes with a set of chaos faults that are defined in a sequence, which helps achieve the desired chaos impact on the target resources. You can sign in to the Harness platform and leverage the interactive chaos access to define your chaos experiments and target the infrastructure. You can also monitor the experiments during their execution.
+**Harness control plane** is the single source to collaboratively create, schedule, and monitor chaos experiments. It comes with a set of chaos faults that are defined in a sequence, which helps achieve the desired chaos impact on the target resources. You can sign in to the Harness platform and leverage the interactive chaos infrastructure to define your chaos experiments and target the infrastructure. You can also monitor the experiments during their execution.
```
Comment (neelanjan00) [inline docs/chaos-engineering/technical-reference/architecture.md:21]:
```suggestion
**Harness control plane** is the single source to collaboratively create, schedule, and monitor chaos experiments. It comes with a set of chaos faults that are defined in a sequence, which helps achieve the desired chaos impact on the target resources. You can sign in to the Harness platform and leverage the interactive UI dashboard to enable chaos access in your infrastructure, define your chaos experiments and target the infrastructure. You can also monitor the experiments during their execution.
```

File: docs/chaos-engineering/technical-reference/architecture.md [hunk 2]
```diff
@@ -39,11 +39,11 @@ Enterprise chaos hub comes out of the box with HCE and provides many faults and
 
 #### Authentication server
```
Comment (neelanjan00) [inline docs/chaos-engineering/technical-reference/architecture.md:40]: Auth server is no more there, we just use Harness auth now. We can remove this entirely.

File: docs/chaos-engineering/technical-reference/architecture.md [hunk 3]
```diff
@@ -39,11 +39,11 @@ Enterprise chaos hub comes out of the box with HCE and provides many faults and
...
 
-The authentication server is a Golang microservice responsible for authorizing and authenticating the requests received from chaos access. It also helps manage users and their projects. It helps in user and project creation, user login, password resets, updating of user information, and managing project-related operations.
+The authentication server is a Golang microservice responsible for authorizing and authenticating the requests received from chaos infrastructure. It also helps manage users and their projects. It helps in user and project creation, user login, password resets, updating of user information, and managing project-related operations.
 
 #### Backend server
```
Comment (neelanjan00) [inline docs/chaos-engineering/technical-reference/architecture.md:44]:
```suggestion
#### Chaos manager
```
Backend server is called chaos manager now

File: docs/chaos-engineering/technical-reference/architecture.md [hunk 4]
```diff
@@ -39,11 +39,11 @@ Enterprise chaos hub comes out of the box with HCE and provides many faults and
...
 
-The backend server is a GraphQL-based Golang microservice that serves the requests received from the chaos access either by querying the database for relevant information or fetching information from the execution plane.
+The backend server is a GraphQL-based Golang microservice that serves the requests received from the chaos infrastructure either by querying the database for relevant information or fetching information from the execution plane.
```
Comment (neelanjan00) [inline docs/chaos-engineering/technical-reference/architecture.md:46]:
```suggestion
The chaos manager is a GraphQL-based Golang microservice that serves the requests either by querying the database for relevant information or fetching information from the execution plane.
```

**Review by neelanjan00 — CHANGES_REQUESTED** [review]

File: docs/chaos-engineering/technical-reference/architecture.md (hunk 1 above)
Comment (SmritiSatya) [reply docs/chaos-engineering/technical-reference/architecture.md:21]: We don't use 'Chaos Access' anymore, no? @neelanjan00

Comment (neelanjan00) [reply docs/chaos-engineering/technical-reference/architecture.md]: chaos access is merely a verb, we can use it as such. We are only prohibiting the use of chaos access as a noun. But we can use what you suggested as well.

**Pull request merged**